src/index.js

```diff
@@ -44,9 +54,7 @@ export default (target, safeOverJump = false) => {
     const lastKey = keys.pop();
     let tmp = target;
     keys.forEach(key => {
-      if (tmp[key] == '__proto__' || tmp[key] == 'prototype' || tmp[key] == 'constructor') {
-        continue;
```

Comment:
`continue` should be replaced with `return` (I guess), since it is inside a `callback` rather than a loop.